### PR TITLE
Fixes experimentally cobbisected techwebs, and prevents it from happening again

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
fix: fixed experimental dissection techweb node
ci: travis will now fail tests on branches with invalid nodes
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -97,13 +97,13 @@
...
 	export_price = 4000
-  
+
 /datum/techweb_node/exp_surgery
 	id = "exp_surgery"
 	display_name = "Experimental Surgery"
 	description = "When evolution isn't fast enough."
 	prereq_ids = list("adv_surgery")
-	design_ids = list("surgery_pacify","surgery_vein_thread","surgery_muscled_veins","surgery_nerve_splice","surgery_nerve_ground","surgery_ligament_hook","surgery_ligament_reinforcement","surgery_viral_bond", "surgery_heal_combo_upgrade", "surgery_exp_dissection")
+	design_ids = list("surgery_pacify","surgery_vein_thread","surgery_muscled_veins","surgery_nerve_splice","surgery_nerve_ground","surgery_ligament_hook","surgery_ligament_reinforcement","surgery_viral_bond", "surgery_heal_combo_upgrade")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/research/techweb/all_nodes.dm]: https://github.com/tgstation/tgstation/blob/master/code/modules/research/designs/medical_designs.dm#L586

wut

Comment (vuonojenmustaturska) [reply code/modules/research/techweb/all_nodes.dm]: that's not a techweb node

Comment (ExcessiveUseOfCobblestone) [reply code/modules/research/techweb/all_nodes.dm]: Right... It's a design ID that goes in the design ID list (that you're removing here) unless i'm missing something.

Comment (vuonojenmustaturska) [reply code/modules/research/techweb/all_nodes.dm]: `## WARNING: Invalid research design with ID surgery_exp_dissection detected in node Experimental Surgery[exp_surgery] removed. in code/controllers/subsystem/research.dm at line 176 src: Research usr: .`

Comment (vuonojenmustaturska) [reply code/modules/research/techweb/all_nodes.dm]: the id is `"surgery_ext_dissection"`

Comment (vuonojenmustaturska) [reply code/modules/research/techweb/all_nodes.dm]: oh i see what you did

**Comment by ExcessiveUseOfCobblestone** [comment]
based copypasting

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Pull request merged**